pages/index.js

```diff
@@ -237,128 +237,150 @@ const FooterContent = styled(Content)`
-
-      <CompanyLogo bottom="0.1rem" height="2.25rem">
-        <InVisionLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.3rem" height="2.75rem">
-        <KiwiComLogo />
-      </CompanyLogo>
-
-      <CompanyLogo>
-        <LegoLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="1rem" height="4rem">
-        <MagicLeapLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="-0.15rem" height="1.3rem">
-        <MoleskineLogo />
-      </CompanyLogo>
-
-      <CompanyLogo>
-        <PatreonLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.15rem">
-        <RedditLogo />
-      </CompanyLogo>
-
-      <CompanyLogo>
-        <SesameLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.4rem" height="3rem">
-        <ShpockLogo />
-      </CompanyLogo>
-
-      <CompanyLogo>
-        <TargetLogo />
-      </CompanyLogo>
-
-      <CompanyLogo height="1.7rem">
-        <TicketmasterLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.2rem">
-        <TinderLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.15rem" height="2.35rem">
-        <TypeformLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.3rem" height="2.1rem">
-        <VimeoLogo />
-      </CompanyLogo>
-
-      <CompanyLogo height="1.8rem">
-        <VogueLogo />
-      </CompanyLogo>
+const SortedLogos = ({ children }) => {
+  // sorting logic: the more popular a website, the higher it gets listed
+  const similarWebGlobalRankSortedLogos = React.Children.toArray(children).sort(
+    (a, b) => a.props.similarWebGlobalRank - b.props.similarWebGlobalRank
+  );
 
-      <CompanyLogo bottom="0.7rem" height="2.5rem">
-        <XingLogo />
-      </CompanyLogo>
-    </>
+  return (
+    <UsersWrapper>
+      {similarWebGlobalRankSortedLogos}
+      {similarWebGlobalRankSortedLogos}
+    </UsersWrapper>
   );
 };
 
+const Logos = [
+  <CompanyLogo bottom="0.3rem" height="2.5rem" key="https://www.airbnb.com" similarWebGlobalRank={20}>
+    <AirBnBLogo />
+  </CompanyLogo>,
+
+  <CompanyLogo
+    bottom="0.16rem"
+    height="2.25rem"
+    src="/static/artsy-logo.svg"
+    key="https://www.artsy.net"
+    similarWebGlobalRank={17576}
+  >
+    <ArtsyLogo />
+  </CompanyLogo>,
+
+  <CompanyLogo height="1.75rem" key="https://www.atlassian.com/" similarWebGlobalRank={2505}>
+    <AtlassianLogo />
+  </CompanyLogo>,
+
+  <CompanyLogo key="https://auth0.com" similarWebGlobalRank={5628}>
+    <Auth0Logo />
+  </CompanyLogo>,
+
+  <CompanyLogo key="https://www.bbc.com/" similarWebGlobalRank={114}>
+    <BBCLogo />
+  </CompanyLogo>,
+
+  <CompanyLogo bottom="-0.2rem" height="1.75rem" key="https://www.bloomberg.com" similarWebGlobalRank={507}>
+    <BloombergLogo />
+  </CompanyLogo>,
+
+  <CompanyLogo bottom="0.75rem" height="3rem" key="https://www.blue-tomato.com" similarWebGlobalRank={21322}>
+    <BlueTomatoLogo />
+  </CompanyLogo>,
+
+  <CompanyLogo bottom="-0.2rem" key="http://casperjs.org/" similarWebGlobalRank={1287463}>
```

Comment:
This is actually `casper.com`, which should have a much higher rank!

Reply:
thx, updated it.